meta-lmp-bsp/conf/machine/imx8mp-lpddr4-evk-sec.conf

```diff
@@ -0,0 +1,10 @@
+#@TYPE: Machine
+#@NAME: NXP i.MX 8M Plus EVK with LPDDR4 Secure
+#@SOC: i.MX8MP
+#@DESCRIPTION: Machine configuration for NXP i.MX 8M Plus Evaluation Kit with LPDDR4
+#              with secure boot enabled by default
+
+require conf/machine/imx8mp-lpddr4-evk.conf
+
+KMACHINE = "imx8mp-lpddr4-evk"
+MACHINEOVERRIDES .= ":imx8mp-lpddr4-evk:imx8mp-lpddr4-evk-sec"
```

Comment:
Can you apply change following change?

```
diff --git a/meta-lmp-bsp/conf/machine/imx8mp-lpddr4-evk-sec.conf b/meta-lmp-bsp/conf/machine/imx8mp-lpddr4-evk-sec.conf
index 30e8508b..781e0616 100644
--- a/meta-lmp-bsp/conf/machine/imx8mp-lpddr4-evk-sec.conf
+++ b/meta-lmp-bsp/conf/machine/imx8mp-lpddr4-evk-sec.conf
@@ -4,7 +4,8 @@
 #@DESCRIPTION: Machine configuration for NXP i.MX 8M Plus Evaluation Kit with LPDDR4
 #              with secure boot enabled by default

+MACHINEOVERRIDES =. "imx8mp-lpddr4-evk:"
+
 require conf/machine/imx8mp-lpddr4-evk.conf

 KMACHINE = "imx8mp-lpddr4-evk"
-MACHINEOVERRIDES .= ":imx8mp-lpddr4-evk:imx8mp-lpddr4-evk-sec"
```

This will have to be changed for the other machines as well, but this fixes a duplication in MACHINEOVERRIDES that causes weird issues during override parsing.

Before:
`MACHINEOVERRIDES="aarch64:armv8a:imx:use-nxp-bsp:imx-boot-container:mx8:mx8m:imxdrm:imxvpu:imxgpu:imxgpu2d:imxgpu3d:mx8mp:imx8mp-lpddr4-evk-sec:imx8mp-lpddr4-evk:imx8mp-lpddr4-evk-sec"`

After:
`MACHINEOVERRIDES="aarch64:armv8a:imx:use-nxp-bsp:imx-boot-container:mx8:mx8m:imxdrm:imxvpu:imxgpu:imxgpu2d:imxgpu3d:mx8mp:imx8mp-lpddr4-evk:imx8mp-lpddr4-evk-sec"`